mailpoet/tests/acceptance/Settings/TransactionalEmailsCest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use MailPoet\Test\DataFactories\Settings;
+
+class TransactionalEmailsCest {
+
+  public function sendTransactionalEmailFallback(\AcceptanceTester $i) {
```

Comment:
Out of curiosity: Would this test succeed even with MailPoet deactivated? What I mean, is our acceptance test server configured in a way, that it would send emails with `wp_mail()` when MailPoet is deactivated? Because in that case I wonder about the meaningfulness of this test :thinking:

Reply:
Good question! It is configured to catch emails send by the PHP mail function (including wp_mail()) by MailHog. Actually, we added it just recently when we switched integration tests to the same infra (https://github.com/mailpoet/wordpress-images/pull/34).
To be sure I tested it and the test passes even when MailPoet is deactivated. 

But! I see that I forgot to ensure in the test that MailPoet is set to send transactional emails. 🤦 I need to add it.

So the goal of the test is that even though the MailPoet sending is misconfigured (it emulates also being blocked by MSS for spam reasons) the transaction emails are still delivered.

Reply:
I've just realized that also the name `withMisconfiguredSendingMethodSmtpMailhog` is not correct. Because it is not Mailhog anymore it is just misconfigured and set to some non-extisting server. Let me update it.